source/_components/sensor.discogs.markdown

```diff
@@ -30,8 +30,14 @@ To enable this sensor, add the following lines to your `configuration.yaml`:
 sensor:
   - platform: discogs
     token: YOUR_TOKEN
+    monitored_conditions:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.